README.md

```diff
@@ -276,6 +277,42 @@ client.links.getInfo({ url: "https://www.jet.com" })
   .catch(handleError);
 ```
 
+### Transactions
+
+
+##### All
+Transactions are a paged resource.  The response object will contain properties `meta.next` and `meta.previous` which can be supplied to subsequent invocations of `#all` to fetch additional results.
+
+`#all` accepts an optional second parameter, `options` which may define the follow keys to narrow results:
+
+###### options
+
+* `cursor`: An API cursor to fetch a specific set of results
+* `start`: An ISO-8601 datetime string to filter only transactions after `start`
+* `end`: An ISO-8601 datetime string to filter only transactions before `end`
+* `time_field`: Time field to filter by: created_date (default) or modified_date
```

Comment:
Recommend make the relationship of this field to `start` and `end` explicit.

Reply:
Done